evap/static/scss/_utilities.scss

```diff
@@ -79,3 +79,9 @@ a.no-underline:hover {
 .break-spaces {
     white-space: break-spaces;
 }
+
+@for $i from 0 through 100 {
+    .width-percent-#{$i} {
+        width: $i * 1% !important;
+    }
+}
```

Comment:
can we reuse bootstraps `w-...` class naming (like `w-100`)?

Reply:
@niklasmohrin also suggested this before. I intentionally did not use bootstrap's scheme to
* make sure the names don't collide if bootstrap ever adds w-X classes
* indicate that our unit is "percent", which is rarely the case for other bootstrap helper classes

Not too strong of an opinion on that, if you both prefer it, we can also change it.